src/Tribe/Featured_Events.php

```diff
@@ -0,0 +1,83 @@
+<?php
+class Tribe__Events__Featured_Events {
+	const FEATURED_EVENT_KEY = '_tribe_featured';
+
+	protected $query_helper;
+	protected $permalinks_helper;
+
+	/**
+	 * @internal
+	 *
+	 * @return Tribe__Events__Featured_Events__Query_Helper
+	 */
+	public function query_helper() {
```

Comment:
Are this and `permalinks_helper` good candidates for trying out the `tribe_singleton()` stuff?